plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -155,8 +155,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                     SUPPORTS_RENAME_FIELD,
                     SUPPORTS_RENAME_SCHEMA,
                     SUPPORTS_SET_COLUMN_TYPE,
-                    SUPPORTS_TOPN_PUSHDOWN,
-                    SUPPORTS_TRUNCATE -> false;
+                    SUPPORTS_TOPN_PUSHDOWN -> false;
```

Comment:
this shouldn't use enhanced switch (so that single item modification is a single line change, not two)